src/include/libutil.h

```diff
@@ -196,6 +196,14 @@ enum vnode_sharing str_to_vnode_sharing(char *vn_str);
  */
 char *pbs_strcat(char **strbuf, int *ssize, char *str);
 
+/*
+ * like strcpy, but returns pointer to end of copied data
+ * useful for chain copies instead of sprintf which is very
+ * slow
+ * 
+ */
+char *pbs_strcpy(char *dest, const char *src);
```

Comment:
would calling it pbs_strcat() make more sense.

Reply:
that would be confusing since strcat actually walks the dest string to its end, and this "expects" the end of the dest char to be provided. The only difference with actually strcpy is the return value. Besides, we already have a pbs_strcat()